### PR TITLE
SEO-173273 - WinUI Searching image changes

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @MallikaRK ,
Worked on image alt text and renamed images.

Regards,
Sharly A.

### CODE REVIEW

File: winui/ComboBox/Searching.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ comboBox.TextMemberPath = "ID";
 
 For e.g. After typing `4` in selection box.
 
-![WinUI ComboBox text searching based on TextMemberPath](Searching_images/winui-combobox-textmemberpath-searching.png)
+![TextMemberPath searching editable mode in WinUI ComboBox](searching_images/winui-combobox-textmemberpath-searching-editable-mode.png)
```
Comment (MallikaRK) [inline winui/ComboBox/Searching.md:52]: Revert your changes in all places. Existing alt text and image fine seems fine, change the casing  in Image Folder name.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: winui/ComboBox/Searching.md (hunk 1 above)
Comment (Sharly-Atieno) [reply winui/ComboBox/Searching.md]: Reverted

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**